body.tex

```diff
@@ -692,7 +700,98 @@ \subsection{Reporting Cycle}
+
+The cost impact analysis of the Rubin Operations risk register informs the level of contingency reserve needed.
+Residual (post-mitigation) cost exposure for each risk is computed by taking the  product of the residual probability of the risk being realized (in any given year) and the likely annual cost of addressing that risk.
+The total cost exposure is then the sum, over all risks in the register, of these products.
+While Rubin holds a number of potentially very costly risks, these are generally very unlikely to occur.
+
+Currently, the total cost exposure, and hence the needed level of contingency reserve, is expected to be approximately 15\% of the total annual operating budget at each operations partner.
+Included in the risk register is the risk of federal funding allocations being reduced, with a likely cost (if realized in any given year) of some \$1.5M per operations partner.
+(At SLAC, for example, this would cover about 7\% of the operating expense -- enough to pay the labor bills during a 1 month funding allocation delay.)
+
+Reserve maintenance is accounted for in our federal budget requests, either as an explicit difference between cost and budget on the DOE side, or as line by line allowances for increases in cost on the NSF side.
+At both operations partners carry forward is maintained from one year's plan to the next, with it being spent down to zero either during the post-operations period (at SLAC), or before the end of each 5-year renewal period at NOIRLab.
+Each year a plan is described in the NOIRLab POP for how carry forward is allocated.
+(The restriction on carrying forward funds between renewal periods means that the level of reserve held at NOIRLab varies significantly over the course of the 5 years.
+Risks that are realized early in the 5-year period are more likely to need supplemental funding from NSF's centrally-held reserves than those that are realized later on, when the carry-forward has been built up.
+It may be possible to purposefully hold back scope so as to build up the carry-forward quickly at the start of a renewal period.)
+
+Requests beyond target (RBTs) are typically made by the departments in order to address risks and/or opportunities as they are realized.
+An RBT, if approved, leads to a draw on the reserve held at one of the operations partners.
+(At NOIRLab, carry-forward use must be declared a year in advance in the POP: this is why the sandboxing process includes a request that the operations teams pitch RBTs for the coming year at that stage.)
+The RBT itself is a Jira ticket submitted by a department's management team on behalf of the team that is addressing the risk.
+Review is by the Directorate team, as budget holders.
+The spending associated with an approved RBT is not captured in a change to the labor or non-labor plan (which together define the baseline spending plan), but instead is tracked as a variance relative to those plans.
```

Comment:
ah so maybe my comment about CCB for RBT is no relevant. I don't think i fully understand the RBT process, does it start proposed as a JIRA ticket and only if granted goes into the CCB process or something like that? - just feel it needs clarification

Reply:
Sure. This is vague because I am attempting to design a specific change process in the absence of the more general change process. So I am hazarding a guess that it will make most sense for teh RBT itself to be a Jira ticket (what else could it be?) and that its processing will be CCB-like. I think we need to establish the CCB process after the JOR and before May, so that we are set up to accept and process RBTs in June.